back/appsync/serverless.yml

```diff
@@ -121,7 +121,7 @@ custom:
     number: 10
   pythonRequirements:
     dockerizePip: non-linux
-    pythonBin: /home/orion/envs/idea36/bin/python
+    pythonBin: python3
```

Comment:
changing one hardcode to other :) not good. rather make it env variable than

Reply:
For your env you can just make a python3 symlink in on of PATH folders. Probably we should create appropriate readme file with details.

Reply:
why we need to do that if u can make this into env variable? And symlink is not something that will work properly for example on windows os

Reply:
Sure env variable is better, should we make a new ticket for this?